paraview/xmls/MergeTreeAutoencoder.xml

```diff
@@ -0,0 +1,653 @@
+          <Entry value="0" text="Adam"/>
+          <Entry value="1" text="SGD"/>
+          <Entry value="2" text="RMSprop"/>
+      </EnumerationDomain>
+        <Documentation>
+          Choose the optimizer method.
+        </Documentation>
+      </IntVectorProperty>
+      
+      <DoubleVectorProperty
+      name="GradientStepSize"
+      command="SetGradientStepSize"
+      label="Gradient Step Size"
+      number_of_elements="1"
+      default_values="0.1">
+        <Documentation>
+          Choose the step use for gradient descent.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="Beta1"
+      command="SetBeta1"
+      label="Beta1"
+      number_of_elements="1"
+      default_values="0.9"
+      panel_visibility="advanced">
+        <Documentation>
+          Coefficients used for computing running averages of gradient and its square.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="Beta2"
+      command="SetBeta2"
+      label="Beta2"
+      number_of_elements="1"
+      default_values="0.999"
+      panel_visibility="advanced">
+        <Documentation>
+          Coefficients used for computing running averages of gradient and its square.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="ReconstructionLossWeight"
+      command="SetReconstructionLossWeight"
+      label="ReconstructionLossWeight"
+      number_of_elements="1"
+      default_values="1.0"
+      panel_visibility="advanced">
+        <Documentation>
+          Weight of the reconstruction loss in the optimization.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="TrackingLossWeight"
+      command="SetTrackingLossWeight"
+      label="TrackingLossWeight"
+      number_of_elements="1"
+      default_values="0.0"
+      panel_visibility="advanced">
+        <Documentation>
+          Weight of the tracking loss in the optimization.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="MetricLossWeight"
+      command="SetMetricLossWeight"
+      label="MetricLossWeight"
+      number_of_elements="1"
+      default_values="0.0"
+      panel_visibility="advanced">
+        <Documentation>
+          Weight of the metric loss in the optimization.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="ClusteringLossWeight"
+      command="SetClusteringLossWeight"
+      label="ClusteringLossWeight"
+      number_of_elements="1"
+      default_values="0.0"
+      panel_visibility="advanced">
+        <Documentation>
+          Weight of the clustering loss in the optimization.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="ClusteringLossTemperature"
```

Comment:
to discuss.

Reply:
thanks for providing a little more explanation, including the name of the term in the paper (and a reference to the corresponding section).

Reply:
Done in the next commit!